src/method/method_invoke_group.cpp

```diff
@@ -198,8 +206,15 @@ namespace cubmethod
 	  }
 
 	error = m_method_vector[i]->get_return (m_thread_p, arg_base, m_result_vector[i]);
+	if (m_rctx->is_interrupted ())
+	  {
+	    error = m_rctx->get_interrupt_reason ();
+	  }
+
```

Comment:
If there is an interrupt, is it correct to ignore the return value of m_method_vector[i]->get_return() and substitute the error variable for the reason of the interrupt?
If the return value of m_method_vector[i]->get_return() is an error, how about handling it first and then checking whether there is an interrupt?

Reply:
Yes, it is correct. In the interrupt situation, the return value of a method_invoke_group does not have to be handled, and only **the interrupt reason of the runtime context should be set as an error and returns the reason code to the client directly**.
For this reason, the error code corresponding to the reason for the interrupt has a higher priority than other execution error codes, so if it is in the interrupt, there is no need to handle it.